project_resource_calendar/views/instrument_view.xml

```diff
@@ -38,7 +38,7 @@
                     <group>
                         <group>
                              <field name="room_id"
-                                       attrs="{'required':[('category_type','=','consumable'),('category_type','=','instrument')], 'invisible':[('category_type','in',['equipment','service'])]}"/>
+                                       attrs="{'required':[('category_type','=','equipment'),('category_type','=','instrument')], 'invisible':[('category_type','in',['consumable','service'])]}"/>
```

Comment:
why not  attrs="{'required':[('category_type','in',['equipment','instrument'])],

Reply:
@rimbendhaou Done